include/envoy/server/lifecycle_notifier.h

```diff
@@ -26,6 +26,11 @@ class ServerLifecycleNotifier {
      */
     PostInit,
 
+    /**
+     * All workers have started.
+     */
+    WorkerStarted,
```

Comment:
This is not used? If this is needed `WorkersStarted`

Reply:
It's used [here](https://github.com/envoyproxy/envoy/pull/14319/files/871c415268f7a712fbb91fd9d7bccacafa3e0557#diff-d367e857aadddf9af71c8dfd9abfb3fc02ba147db56e1fc44ad41f60d1e9dd74R379).

Reply:
I see. My preference would be to not make prod changes like this just for tests. If you need synchronization hooks can you use https://github.com/envoyproxy/envoy/blob/master/source/server/listener_hooks.h instead? Thank you.

/wait